advice_animal/cli.py

```diff
@@ -104,7 +100,13 @@ def main(
     LOG.info("Using settings %s", ctx.obj)
 
 
-@main.command()
+@main.group()
```

Comment:
This seems to include changes from https://github.com/advice-animal/advice-animal/commit/55816a6db1ebf48a87227db9bed15ed6dca3b3c7 as well, I'm not sure why.

Reply:
I rebased with main.